Services/COPage/classes/class.ilPCVerification.php

```diff
@@ -11,17 +11,17 @@
  */
 class ilPCVerification extends ilPageContent
 {
+    private php4DOMElement $vrfc_node;
+    protected ilObjUser$user;
     /**
-     * @var ilObjUser
+     * @var php4DOMDocument
```

Comment:
IMO this property can be completely deleted because it is derive from `ilPageContent`.